### ISSUE TITLE
Elkapod base stability check part 1

### ISSUE THREAD

**Issue by VisteK528** [HexapodBionik/MotionPlanning#16]
# First development phase of Elkapod base stability check component
The main objective for this phase is to develop a simple stability check module that evalutates the robot's stability based on foot contact points and predefined center of mass.

All contact points coordinates and center of mass are described relative to the robot's base coordinate system (for now it's a center of 0.2x0.3x0.05 cuboid. To make things simpler for now the solver should assume that the roll and pitch angles are equal to 0 (connection with IMU will be added in the next task).

The module should support stability check for all possible numbers of legs placed on the ground (still, tripoid gait, wave gait and the situation where 5 legs are used for support and the sixth limb is used as a primitive manipulator).

The class should include one method that returns the margin of stability and the second and main method for stability check should return the bool depending on the priorly defined stability_margin argument of the class. (It should be possible to change this parameter using setter and getter, but for the initialization it should be also passed in the class' constructor). Both of them should be public.

All files should be placed in separate folder named `stability_check` in `MotionPlanning` repository and developed on a separate branch named `stability_check`

It is preferred to use numpy arrays for input instead of requesting multiple float inputs for a class method.

In conclusion, things to be done:
- [x] Python module (or modules for clarity) containting StabilityCheck class 
- [x] Short, but precise documentation showing how the algorithm works (eg. geometrical interpretation)
- [x] Unit tests in subfolder (min. 10 unit tests)

**Comment by VisteK528** [comment]
Move the documentation to specific folder in the ElkapodDocumentation repository as we discussed yesterday.
